script.js

```diff
@@ -1,12 +1,125 @@
-//You can edit ALL of the code here
+const rootElem = document.getElementById("root");
```

Comment:
If this is the same every time it's best to put it in HTML and then use `findElementById()` to access it from JS

Reply:
Hi Chris, I am not sure if I understood correctly. Do you advise findElementById() over getElementById()?